src/main/java/com/example/manager/controller/UserController.java

```diff
@@ -0,0 +1,143 @@
+package com.example.manager.controller;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.Date;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import org.springframework.stereotype.Service;
+
+import com.example.manager.repository.UserRepository;
+
+import com.example.manager.domain.User;
+
+import org.springframework.security.crypto.password.PasswordEncoder;
+
+@Service
+public class UserController {

```

Comment:
Since it is a service, maybe adjust the name of this class to "UserService" - since Controllers in Spring are the entry point for Rest Calls, which then calls any Service to procede with the implementation